src/Native/LibMFWrapper/CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+project (FactorizationMachineNative)
```

Comment:
>FactorizationMachineNative [](start = 9, length = 26)

Nope

Reply:
It becomes LIBMFNative now. Thanks.

---
In reply to: [225360940](https://github.com/dotnet/machinelearning/pull/1263#discussion_r225360940) [](ancestors = 225360940)